buildtools/compile-catalog.js

```diff
@@ -8,30 +6,25 @@ function main(inputs) {
   var compiler = new Compiler({format: 'json'});
 
   var contents = [];
-  async.eachSeries(inputs, function(input, cb) {
+  inputs.forEach(function(input) {
+    // ignore un existing files
     fs.exists(input, function(exists) {
       if (exists) {
-        fs.stat(input, function(error, stats) {
-          if (stats.size != 0) {
-            fs.readFile(input, {encoding: 'utf-8'}, function(err, content) {
-              if (!err) {
-                contents.push(content);
-              }
-              cb(err);
-            });
-          } else {
-            process.stdout.write("{}");
+        fs.readFile(input, {encoding: 'utf-8'}, function(err, content) {
+          if (!err) {
+            contents.push(content);
+            if (contents.length == inputs.length) {
```

Comment:
`contents.length` is always a number so there is no need to use the type-safe equals.